portfolio/src/main/webapp/script.js

```diff
@@ -87,6 +109,15 @@ function createH4Element(text) {
   return h4element;
 }
 
+function createAElement(text, url) {
+  const aElement = document.createElement('a');
+  var textNode = document.createTextNode(text);
+  aElement.appendChild(textNode);
```

Comment:
supernit: you could alternative just set `aElement.innerText = text`

Reply:
Got it!